tools/macos/README.md

```diff
@@ -1,8 +1,76 @@
-#docker-machine:
+# How to setup a Mac host for OpenWhisk
 
-* one time setup
+One way to develop or deploy OpenWhisk on a Mac is to use [docker-machine](https://docs.docker.com/machine/install-machine/) which runs the Docker daemon inside a virtual machine accessible from the Mac host.
+
+# Prerequisites
+
+The following are required to build OpenWhisk. The easiest way to install the following is using [Homebrew](http://brew.sh/) with [Cask](https://github.com/caskroom/homebrew-cask). You can copy the entire section below and paste into your terminal to run all of the steps required.
```

Comment:
This paragraph implies that there is a list of depencencies, while there is none. So I'ld rather add a list of all the things to install + links followed by our set of commands for users of `brew`, or rewrite the paragraph as:

The following commands will install all prerequisites needed to build and deploy OpenWhisk locally. You need to have [Homebrew](http://brew.sh/) with [Cask](https://github.com/caskroom/homebrew-cask) installed for them to work. Copy the entire section below and paste it into your terminal to run all of the steps required.